src/index.html

```diff
@@ -34,6 +34,20 @@
 					}
 				};
 				cmp.config = {
+					/*
+					theme: {
+						overlayBackground: 'rgba(0, 0, 0, 0.84)',
```

Comment:
I'd rather these were more explicit and had `Color` in the name of color variables.  Should make it clearer if we add sizes or other values in the future.